csharp/src/iceboxnet/ServiceManager.cs

```diff
@@ -398,9 +394,11 @@ async Task StartedAsync(IServiceObserverPrx observer, string[] services)
             }
         }
 
-        private void ServicesStopped(ICollection<string> services)
+        private void ServiceStopped(string name) => ServicesStopped(ImmutableList.Create(name));
+
+        private void ServicesStopped(IEnumerable<string> services)
```

Comment:
I think `IReadOnlyList<string>` makes more sense here and keep `services.Cout > 0` as before.

Reply:
Reverted back to `ICollection<string>`.